src/BadgeBatch/SkillFileParser.php

```diff
@@ -11,19 +13,20 @@ class SkillFileParser
      */
     public function parse($content)
     {
+        $parser = new Parser();
         $skills = [];
-        foreach (explode("\n", $content) as $line) {
-            $line = trim($line);
-            if (! $line) {
-                continue;
-            }
 
-            $exp = array_filter(array_map('trim', explode(':', $line, 2)));
-            if (count($exp) != 2) {
+        // todo might throw Symfony\Component\Yaml\Exception\ParseException
+        //      do we want to catch this here or handle further up
```

Comment:
I vote for catching here and simply ignoring anything that is in invalid format.